content/articles/getting-started-with-question-answering-using-bert/index.md

```diff
@@ -0,0 +1,157 @@
+
+### What is BERT
+Bidirectional Encoder Representations from Transformers (BERT) is a natural language processing model that uses transformers to accomplish a wide variety of NLP tasks. Some of the tasks where this model performs well include Question Answering, [Natural Language Inference](http://nlpprogress.com/english/natural_language_inference.html), and [Named Entity Recognition](https://en.wikipedia.org/wiki/Named-entity_recognition). Please refer to this [article](/engineering-education/introducing-gpt3/) to learn more about BERT and how it compares to other natural language processing models such as GPT-3.
+
+This model was originally presented by Jacob Devlin and his research team at Google in 2018.
+
+### How to install the BERT model and import its dependencies
+We will be using the pre-built BERT model from TensorFlow.js for this build. Pre-trained models make it easy to get started with huge models without getting involved with many setup and training.
+
+We will be installing three packages. `@tensorflow/tfjs` to install tensorflow.js library, `@tensorflow-models/qna` to download the question-answering model, and `react-loader-spinner` to give us a nice loading interface as our BERT model is being downloaded. It is quite a large model. Thus, a nice loading interface would come in handy.
+
+```js
+npm i @tensorflow/tfjs @tensorflow-models/qna react-loader-spinner
+```
+After successful installation, you can start up the react app to see if everything is okay. We start react using the `npm start` command as shown below:
+
+```js
+npm start
+```
+If you see a "Hello World!" on the screen, we are good to go. It means that the app is functioning well. You can stop the app on the terminal by pressing `CTRL+C` simultaneously. We can now go ahead and import the required dependencies.
+
+```js
+import * as tf from "@tensorflow/tfjs";
+import * as qna from "@tensorflow-models/qna";
+import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
+import Loader from "react-loader-spinner";
+import { Fragment } from 'react';
+```
+Our first import allows us to refer to the tensorflow.js library in our code as `tf`. The second import allows us to refer to the BERT model inside our code as `qna`. The third and fourth import imports the css style for our loader that we mentioned above. Lastly, we've imported `Fragment`, which allows us to return multiple elements without adding extra nodes to the DOM. 
+
+The next step involves setting up our references and state hooks. Hooks tend to create a symbolic link to various elements. By default, the React library imports the following hooks; `useRef`, `useEffect`, and `useState`. You can see it at the top on the `App.js` file. `useState` allows us to work with states within our react app. It will be used to store our question and passage. For a detailed explanation about hooks and references in React, please refer to this [documentation](https://reactjs.org/docs/hooks-intro.html)
```

Comment:
```suggestion
The next step involves setting up our references and state hooks. Hooks tend to create a symbolic link to various elements. By default, the React library imports the following hooks; `useRef`, `useEffect`, and `useState`. You can see it at the top on the `App.js` file. `useState` allows us to work with states within our react app. It will be used to store our questions and passage. For a detailed explanation about hooks and references in React, please refer to this [documentation](https://reactjs.org/docs/hooks-intro.html).
```